### DISCUSSION TITLE
Connect Azure SQL server using MFA

### DISCUSSION THREAD

**Discussion by ohadyana** [dbeaver/dbeaver#21244] [General]
Hello,

I am using DBeaver community edition V 23.2.0.202309041200
I am trying to create a connection to our Azure SQL Server using active directory MFA authentication. When doing so the process opens a new tab in my internet browser and I get the message: 'Authentication complete. You can close the browser and return to the application.'
I close this tab and it opens again.
After few attempts I get this message:
![dbeaver](https://github.com/dbeaver/dbeaver/assets/69153131/602efd22-e09d-40b9-82f7-79f436259f49)
If I try to connect with Dbeaver enterprise edition (23.1.0.202306120813) the above works as expected.
Does DBeaver support MFA to azure sql server only for advance editions ?

Thank you

**Comment by LonwoLonwo** [comment]
Hello @ohadyana 

Could you please provide a screen from your Connection Settings -> Driver Settings -> Libraries page?

**Reply by ohadyana** [reply]
Yes of course
![image](https://github.com/dbeaver/dbeaver/assets/69153131/2d0c9c45-7d62-480f-bbdf-3886a8644ba0)
Thanks !

**Reply by LonwoLonwo** [reply]
There are too many files/folders in libraries. Are they important? Maybe it will be better to remove folders and leave only jars?
Also, the red highlighting means that there is no such a file on this path.
Also, maybe you can try to change mssql-jdbc and msal4j versions. Sometimes, it also helps to connect.
https://github.com/dbeaver/dbeaver/issues?q=label%3Ax%3Aazure+is%3Aclosed we had many discussions about it

**Reply by ohadyana** [reply]
Thanks again !
 I removed all the unnecessary folder \ files and add all the files that mention in the next comment (#19802).
Unfortunately the same result happend
The files :
![image](https://github.com/dbeaver/dbeaver/assets/69153131/fa952280-d431-4a89-9ce9-c819fa8a98e2)
Error message:
![image](https://github.com/dbeaver/dbeaver/assets/69153131/c3ed3924-e752-4b29-b87c-0c32f5558fa2)

**Comment by LonwoLonwo** [comment]
Maybe this answer will help: https://github.com/dbeaver/dbeaver/issues/19802#issuecomment-1522083132

**Reply by ohadyana** [reply]
Thanks again :-)
This is the result 
![image](https://github.com/dbeaver/dbeaver/assets/69153131/537eb6b3-4ddd-41ef-a55f-9a120ed26bb4)
